marge/batch_job.py

```diff
@@ -198,6 +182,9 @@ def execute(self):
                     merge_request.source_branch,
                     '%s/%s' % (merge_request_remote, merge_request.source_branch),
                 )
+                # Apply the trailers before running the batch MR
+                self.add_trailers(merge_request)
+                self.push_force_to_mr(merge_request, True, source_repo_url, skip_ci=True)
```

Comment:
It would be nicer to specify keyword args for the boolean, as makes it easier to know what this is doing (i.e. `branch_was_modified=True`)